src/idsk/components/related-content/template.njk

```diff
@@ -0,0 +1,14 @@
+
+<div class="govuk-grid-column-{%- if params.oneThird %}one-third{% else %}two-thirds{% endif %}">
```

Comment:
toto by som prepisal radsej tak {{ 'one-third' if params.oneThird else 'two-thirds' }} je to citatelnejsie